token-lending/program/src/processor.rs

```diff
@@ -818,7 +854,37 @@ fn process_deposit_obligation_collateral(
     let user_transfer_authority_info = next_account_info(account_info_iter)?;
     let clock = &Clock::from_account_info(next_account_info(account_info_iter)?)?;
     let token_program_id = next_account_info(account_info_iter)?;
+    _deposit_obligation_collateral(
+        program_id,
+        collateral_amount,
+        source_collateral_info,
+        destination_collateral_info,
+        deposit_reserve_info,
+        obligation_info,
+        lending_market_info,
+        lending_market_authority_info,
+        obligation_owner_info,
+        user_transfer_authority_info,
+        clock,
+        token_program_id,
+    )
```

Comment:
should have ?

Reply:
I think this doesn't need it
no semicolon means that it's an "expression" rather than a "statement", so rust auto returns the last "expression" return value

Reply:
hmm i guess it prob works either way. not suggesting a ; just a ? but yeh